doc/release-notes-1.2.0.rst

```diff
@@ -0,0 +1,499 @@
+    * :ref:`bootloader` - updated
+    * :ref:`nrf_desktop` - updated
+
+
+* Added or updated documentation for the following libraries:
+
+  * nRF9160:
+
+    * :ref:`lib_aws_iot` - added
+    * :ref:`cloud_api_readme` - added
+    * :ref:`modem_key_mgmt` - added
+    * :ref:`sms_readme` - added
+    * :ref:`supl_client` - added
+    * :ref:`at_cmd_readme` - updated
+    * :ref:`at_cmd_parser_readme` - updated
+    * :ref:`lib_download_client` - updated
+
+  * Bluetooth Low Energy:
+
+    * :ref:`bt_mesh_dk_prov` - added
+    * :ref:`latency_c_readme` - added
+    * :ref:`latency_readme` - added
+    * :ref:`shell_bt_nus_readme` - updated
+
+  * NFC:
+
+    * :ref:`nfc_t4t_cc_file_readme` - added
+    * :ref:`st25r3911b_nfc_readme` - added
+    * :ref:`nfc_t4t_hl_procedure_readme` - added
+    * :ref:`tnep_poller_readme` - added
+    * :ref:`tnep_tag_readme` - added
+
+  * Other:
+
+    * :ref:`doc_bl_crypto` - added
+    * :ref:`doc_bl_validation` - added
+    * :ref:`fprotect_readme` - added
+    * :ref:`lib_dfu_target` - updated
+    * :ref:`doc_fw_info` - updated
+    * :ref:`partition_manager` - updated
+
+
+* Added or updated the following documentation:
+
+  * Getting started:
+
+    * :ref:`gs_installing` - restructured parts of the content
+    * :ref:`gs_programming` - restructured the content and added information about building on the command line
+    * :ref:`gs_modifying` - updated the content and added information about configuring an application
+
+  * User guides:
+
+    * :ref:`ug_nrf5340` - added
+    * :ref:`ug_thingy91` - added
+    * :ref:`ug_ble_controller` - added
+    * :ref:`ug_multi_image` - updated with content that was removed from the Zephyr fork
+
+  * nrfxlib:
+
+    * :ref:`nrfxlib:ble_controller_readme` - updated to match current version of the nRF Bluetooth LE Controller
+    * :ref:`nrfxlib:bsdlib` - extended and restructured the content
+    * :ref:`nrfxlib:mpsl` - added
+
+
+
+Known issues
+************
+
+??? Must be checked and updated
+
+nRF9160
+=======
+
+* Deprecation warning: The nrf_inbuilt_key API in the :ref:`nrfxlib:bsdlib` will be removed in a future release.
+  A replacement library that wraps the AT commands for ``AT%CMNG`` will be available in the |NCS|.
+* The :ref:`asset_tracker` sample might show up to 2.5 mA current consumption in idle mode with ``CONFIG_POWER_OPTIMIZATION_ENABLE=y``.
+* The SEGGER Control Block cannot be found by automatic search by the RTT Viewer/Logger.
+  As a workaround, set the RTT Control Block address to 0 and it will try to search from address 0 and upwards.
+  If this does not work, look in the ``builddir/zephyr/zephyr.map`` file to find the address of the ``_SEGGER_RTT`` symbol in the map file and use that as input to the viewer/logger.
+* nRF91 fails to receive large packets (over 4000 bytes).
+* nrf_connect fails if called immediately after initialization of the device.
+  A delay of 1000 ms is required for this to work as intended.
+
```

Comment:
* The GPS sample will stop working if the if SUPL client library support is enabled and it fails to resolve the SUPL hostname.
As a workaround can a few second delay, using k_sleep(), be inserted after the printf on line 294 in main.c.